test/controllers/area.controller.test.js

```diff
@@ -148,4 +144,159 @@ describe('Area Controller', () => {
       });
     });
   });
+
+  describe('Get all states', () => {
+    const lagosState = AreaFactory.build({ state: 'lagos' });
+    const oyoState = AreaFactory.build({ state: 'oyo' });
+    beforeEach(async () => {
+      await addArea(lagosState);
+      await addArea(oyoState);
+    });
+
+    context('when editor token is used', () => {
+      it('returns array of five states', (done) => {
```

Comment:
Not sure why you used 5 states here:
```suggestion
      it('returns array of two states', (done) => {
```